model/Core/Vocabularies/HashAlgorithm.md

```diff
@@ -18,6 +18,7 @@ practically infeasible to invert.
 
 ## Entries
 
+- adler32: Adler-32 checksum is part of the widely used zlib compression library as defined in [RFC 1950])(https://www.ietf.org/rfc/rfc1950.txt) Section 2.3.
```

Comment:
```suggestion
- adler32: Adler-32 checksum is part of the widely used zlib compression library as defined in [RFC 1950](https://www.rfc-editor.org/info/rfc1950) Section 2.3.
```
 
- Remove extra `)`
- Use https://www.rfc-editor.org/info/rfc1950 to match the style with other existing RFC URLs